torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2129,20 +2133,9 @@ def sample_inputs_svd_lowrank(op_info, device, dtype, requires_grad=False, **kwa
         # we can only use k for q.
         # This issues could be resolved with using a rank-revealing SVD
         # which does not include "zero" singular values.
-        op_kwargs = {
-            'q': k,
-            'M': None
-        }
-
-        # without M specified
-        yield clone_sample(sample, **op_kwargs)
-
-        # now with M
-        # TODO: fix bug in the documentation for svd_lowrank:
-        # M has to be (*, m, n), and not (*, 1, n) as written
-        # in the documentation
-        op_kwargs['M'] = make_tensor((*batch, m, n), dtype=dtype, device=device, requires_grad=requires_grad)
-        yield clone_sample(sample, **op_kwargs)
+        yield SampleInput(a, b, q=k, M=None).with_metadata(output_process_fn_grad=fn)
+        M = make_tensor((*batch, m, n), dtype=dtype, device=device, requires_grad=requires_grad)
+        yield SampleInput(a, b, q=k, M=M).with_metadata(output_process_fn_grad=fn)
```

Comment:
Notice that previously cloned samples were returned. It is likely that cloning is required here as well. Say:
```suggestion
        yield clone_sample(SampleInput(a, b, q=k, M=None)).with_metadata(output_process_fn_grad=fn)
        M = make_tensor((*batch, m, n), dtype=dtype, device=device, requires_grad=requires_grad)
        yield clone_sample(SampleInput(a, b, q=k, M=M)).with_metadata(output_process_fn_grad=fn)
```